### PR TITLE
MAT-6393: Fix condition for unique valuesets with no author

### PR DESCRIPTION
**Pull request by mcmcphillips**
## Terminology Service PR

Jira Ticket: [MAT-0000](https://jira.cms.gov/browse/MAT-0000)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by jkotanchik-SB — DISMISSED** [review]

File: src/main/java/gov/cms/madie/terminology/service/FhirTerminologyService.java [hunk 1]
```diff
@@ -140,10 +140,11 @@ public List<ValueSetForSearch> searchValueSets(String apiKey, Map<String, String
                     ValueSetForSearch.builder()
                         .title(vs.getTitle())
                         .author(
-                            String.valueOf(
-                                vs.getExtensionByUrl(
-                                        "http://hl7.org/fhir/StructureDefinition/valueset-author")
-                                    .getValue()))
+                            Optional.ofNullable(
+                                    vs.getExtensionByUrl(
+                                        "http://hl7.org/fhir/StructureDefinition/valueset   -author"))
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/madie/terminology/service/FhirTerminologyService.java]: I'm seeing some funky whitespace between `valueset` and `-author`.

Comment (mcmcphillips) [reply src/main/java/gov/cms/madie/terminology/service/FhirTerminologyService.java]: I think my cat might have co authored.

**Review by jkotanchik-SB — APPROVED** [review]
